src/components/InputText/InputText.css

```diff
@@ -0,0 +1,14 @@
+.Input-Text{
```

Comment:
Pascal Case needed here too
```suggestion
.InputText{
```

Reply:
@Mubangizi this has been fixed.